Capstone/Feed/HomeCell.h

```diff
@@ -13,6 +13,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property (weak, nonatomic) IBOutlet UIImageView *homeCellImage;
 @property (weak, nonatomic) IBOutlet UILabel *homeCollegeName;
 @property (weak, nonatomic) IBOutlet UILabel *homeCollegeDetails;
+@property (weak, nonatomic) IBOutlet UILabel *homeCollegeLocation;
+@property (weak, nonatomic) NSArray *cellDictionary;
```

Comment:
It'd be great if this array had a type so it's clear what objects go into this cell dictionary! I'm making my best guess — and the fact I have to guess is probably not good

```suggestion
@property (weak, nonatomic) NSArray<String *> *cellDictionary;
```

Also, what's the purpose of the dict? The name is a little vague